patches/chrome-browser-permissions-permission_manager.cc.patch

```diff
@@ -11,9 +11,9 @@ index 7c41959fe6acb8aded0db703e0327c2251c39fe4..f6e6c9e9482a56ac2e3b12ce2609ca2e
    switch (permission) {
      case PermissionType::MIDI:
        return CONTENT_SETTINGS_TYPE_MIDI;
-@@ -129,6 +129,7 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
-     case PermissionType::IDLE_DETECTION:
-       return CONTENT_SETTINGS_TYPE_IDLE_DETECTION;
+@@ -132,6 +132,7 @@ ContentSettingsType PermissionTypeToContentSetting(PermissionType permission) {
+     case PermissionType::PERIODIC_BACKGROUND_SYNC:
+       return CONTENT_SETTINGS_TYPE_PERIODIC_BACKGROUND_SYNC;
      case PermissionType::NUM:
 +    default:
```

Comment:
this doesn't seem like it's needed, maybe we can remove this patch in a followup?

Reply:
Looks like we extended the enum, so without it we get a compilation error:
<pre>
brave-browser\src\brave\chromium_src\chrome\browser\permissions/../../../../../chrome/browser/permissions/permission_manager.cc(92,11):
error: enumeration  value 'AUTOPLAY' not handled in switch [-Werror,-Wswitch]
switch (permission) {
^
1 error generated.
</pre>